python/requirements.txt

```diff
@@ -47,6 +47,7 @@ azure-common
 azure-mgmt-resource
 msrestazure
 boto3
+botocore
```

Comment:
We may need to add this elsewhere.
Given that botocore is a dependency of boto3, why do we need this?

Reply:
I'm using botocore to catch client-side exceptions in cloudwatch_helper.py. I can try using just boto3 client.exceptions instead and avoid importing another dependency in updated commit.

Reply:
I think that would be preferable.

Reply:
Looks like we still need botocore imported to catch error statically defined within botocore package other than ClientError thrown by AWS service exceptions.
In cloudwatch_helper.py, botocore is mainly used to catch WaiterError that wait for cloudwatch agent installation and configuration commands to complete.

Reply:
Got it. If there's a need to make CloudWatch features work with Docker support, the botocore requirement might have to be added [here](https://github.com/ray-project/ray/blob/b5ccee6ad3bf2b4b2fdc6966ddfea965e027030d/docker/ray-deps/Dockerfile).
I'm not entirely sure cc @ijrsvt